scripts/sign-apk.js

```diff
@@ -0,0 +1,15 @@
+
+import { androidHelpers } from 'appium-android-driver';
+import { asyncify } from 'asyncbox';
+import path from 'path';
+import packageJson from '../package.json';
+
+async function main () {
+  // Signs the APK with the default Appium Certificate
+  const adb = await androidHelpers.createADB({});
+  const pathToApk = path.resolve('apks', `appium-uiautomator2-server-v${packageJson.version}.apk`);
+  await adb.sign(pathToApk);
+  await adb.executeApksigner(['verify', '--print-certs', pathToApk]);
```

Comment:
this will fail if apksigner is not installed on the publisher's machine

Reply:
I'll take that out. It was only there to print the cert so that I could verify that it was working.